docs/serving/load-balancing/README.md

```diff
@@ -0,0 +1,30 @@
+You can configure load balancing on Knative to place the activator in the request path to act as a load balancer.
+
+This guide explains how you can configure load balancing for your Knative system using the activator.
+
+## Prerequisites
+- Ensure that load balancing using an ingress gateway (for example, Istio or Kourier) is not enabled.
```

Comment:
We don't disable the ingress (let's remove word gateway, since it's Istio specific) load balancing. It will apply only to distributing requests to activators.

Reply:
@vagababov still awaiting some clarification on this one?

Reply:
@vagababov will I just remove this line? I'd like to get this PR closed since it has been months.